### ISSUE TITLE
Create a readme how to create a standalone serializer / validator / normalizer

### ISSUE THREAD

**Issue by dkarlovi** [sigwinhq/xezilaires-dev#147]
Quick example:

1. add deps
```
composer require sigwin/xezilaires-symfony doctrine/annotations sigwin/xezilaires-spout
```

2. autoload snippet
```json
    "autoload": {
        "psr-4": {
            "Test\\": "."
        }
    },
```

3. model example

```php
<?php

namespace Test;

use Xezilaires\Annotation as XLS;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Serializer\Annotation as Serializer;
use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer;

/**
 * @XLS\Options(header=1,start=2)
 */
class Product
{
    /**
     * @var int
     * @XLS\HeaderReference(header="id")
     * @Assert\Type(type="integer")
     * @Assert\PositiveOrZero()
     */
    public $id;

    /**
     * @var string
     * @XLS\HeaderReference(header="name")
     * @Assert\NotBlank()
     */
    public $name;

    /**
     * @var \DateTimeInterface
     * @XLS\HeaderReference(header="date")
     * @Assert\Date()
     * @Serializer\Context(context={DateTimeNormalizer::FORMAT_KEY: "d.m.Y."})
     */
    public $date;

    /**
     * @var float
     * @XLS\HeaderReference(header="price")
     * @Assert\PositiveOrZero()
     */
    public $price;
}
```

4. example file(s):
[file1.xlsx](https://github.com/sigwinhq/xezilaires-dev/files/7039129/file1.xlsx)
[file2.xlsx](https://github.com/sigwinhq/xezilaires-dev/files/7039130/file2.xlsx)

5. commands

```
vendor/bin/xezilaires validate Test\\Product file*.xlsx
vendor/bin/xezilaires serialize Test\\Product file1.xlsx
```

**Comment by dkarlovi** [comment]
Needs Symfony 6.1 minimum: https://github.com/symfony/symfony/pull/43701

**Comment by dkarlovi** [comment]
Closed in #176.
